hawkbit-dmf/hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpMessageHandlerService.java

```diff
@@ -13,36 +13,23 @@
 
 import java.io.Serializable;
 import java.net.URI;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Don't use .* for import. Only import the classes that are needed.

Reply:
Thanks, was because of the formatter issues.